packages/react-components/src/components/gif.tsx

```diff
@@ -231,20 +239,26 @@ const Gif = ({
         >
             <div style={{ width, height, position: 'relative' }}>
                 <img
+                    className={Gif.imgClassName}
                     src={showGif ? fit : placeholder}
                     style={{ background }}
                     width={width}
                     height={height}
                     alt={getAltText(gif)}
                     onLoad={showGif ? onImageLoad : () => {}}
                 />
-                {showGif ? <AdPill bottleData={bottleData} /> : null}
-                {Overlay && <Overlay gif={gif} isHovered={isHovered} />}
+                {showGif ? (
+                    <>
+                        <AdPill bottleData={bottleData} />
+                        {Overlay && <Overlay gif={gif} isHovered={isHovered} />}
```

Comment:
Don't render the Overlay until the `Gif` component is on the screen. Wondering if an ad can ever have attribution? 🤔 the attribution would cover the pill